src/providers/AppLinkProvider.tsx

```diff
@@ -10,7 +10,7 @@ import React, {
 import { Linking } from 'react-native'
 import queryString from 'query-string'
 import { useSelector } from 'react-redux'
-import { Address } from '@helium/crypto-react-native'
```

Comment:
So is this some deprecated version of `Address` ?

Reply:
Yeah, `Address` was moved to its own package `@helium/address` and it used to be part of the crypto library. When I updated the libraries it needed refactoring